core/src/main/resources/hudson/model/Run/logKeep.jelly

```diff
@@ -27,7 +27,7 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <j:if test="${it.parent.buildDiscarder!=null and it.canToggleLogKeep()}">
+  <j:if test="${it.canToggleLogKeep()}">
```

Comment:
🥇 for remembering to enable `Keep Build Forever` button in the case where a explicit build discarder isn't set for the job